pkg/broker/broker.go

```diff
@@ -1002,9 +1002,9 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
 
 		if async, provision: it should create a Job that calls apb.Provision. And write the output to etcd.
 	*/
-	a.log.Debug(fmt.Sprintf("service_id: %s", req.ServiceID)) // optional
-	a.log.Debug(fmt.Sprintf("plan_id: %s", req.PlanID))       // optional
-	a.log.Debug(fmt.Sprintf("operation:  %s", req.Operation)) // this is provided with the provision. task id from the work_engine
+	a.log.Debugf("service_id: %s", req.ServiceID)
+	a.log.Debugf("plan_id: %s", req.PlanID)
+	a.log.Debugf("operation:  %s", req.Operation) // this is provided with the provision. task id from the work_engine
```

Comment:
Nit: the comment is a little inaccurate, it'll be there with every async op, not just provision.